### DISCUSSION TITLE
Permintaan Izin Mempelajari Source Code

### DISCUSSION THREAD

**Discussion by dhilataha** [wildangunawan/Dataset-Saham-IDX#2] [Q&A] [answered]
Halo, Selamat Malam
Saya Fadhillah Taha mahasiswa akhir Program Studi Ilmu Komputer Universitas Hasanuddin, sedang mengerjakan tugas akhir menggunakan data pergerakan harga saham dalam memprediksi pergerakan harga saham menggunakan pembelajaran jaringan mendalam (Deep Learning). 
Saya mendapatkan dataset dari Kaggle, jika kakak berkenan, apakah saya bisa mendapatkan program kakak untuk mempelajari lebih lanjut bagaimana cara mengumpulkan data dari BEI 
Terima Kasih

**Comment by wildangunawan** [comment, marked as answer]
Halo,

Saya sudah publish source code untuk scrap datanya di repository saya yang lain. Cek di https://github.com/wildangunawan/IDX-Scrapper.

Dalam repository tersebut saya juga sudah menuliskan lisensi dan cara penggunaannya. Silakan coba dilihat ya kak.

Good luck untuk tugas akhirnya!

**Reply by dhilataha** [reply]
ohiya terima kasih banyak kak semoga sukses selalu

**Reply by dhilataha** [reply]
kak, mau tanya kalau misalnya datanya langsung ku unduh dri hasil _collecting_ kakak, itu termasuk data primer atau data sekunder ya? mengingat data yang kakak kumpulkan sepertinya data mentah yang belum diolah
terima kasih banyak

**Reply by wildangunawan** [reply]
Halo,

Untuk itu saya kurang tau ya. Silakan didiskusikan dengan dosen pembimbing kakaknya ya :)

**Reply by dhilataha** [reply]
baik, terima kasih banyak kak

**Reply by steamboatid** [reply]
> Saya sudah publish source code untuk scrap datanya di repository saya yang lain. Cek di https://github.com/wildangunawan/IDX-Scrapper.

ini reponya sudah ilang ya mas?

**Reply by wildangunawan** [reply]
Sudah aku private utk comply dengan peraturan BEI yang baru ya: https://github.com/wildangunawan/Dataset-Saham-IDX?tab=readme-ov-file#penggunaan-untuk-komersial-maupun-non-komersial

**Reply by steamboatid** [reply]
> Sudah aku private utk comply dengan peraturan BEI yang baru ya: https://github.com/wildangunawan/Dataset-Saham-IDX?tab=readme-ov-file#penggunaan-untuk-komersial-maupun-non-komersial

hmmm... terus, bagaimana caranya kalau mau ngintip source nya?
apa bisa dikirimkan private?

**Reply by wildangunawan** [reply]
Pak @steamboatid mohon dibantu kirim email ke gwildan50 (at) gmail.com ya beserta keperluan bapak. Apabila OK, nanti aku invite ke repositorynya secara langsung.

**Reply by steamboatid** [reply]
> Pak @steamboatid mohon dibantu kirim email ke gwildan50 (at) gmail.com ya beserta keperluan bapak. Apabila OK, nanti aku invite ke repositorynya secara langsung.

email sent. terima kasih.
